idea/idea-completion/testData/keywords/InFunctionTypePosition.kt

```diff
@@ -2,5 +2,5 @@ fun foo() {
     val test : <caret>
```

Comment:
Could you please add a test for completion in function argument type parameter? 

Like this: 

```kt
fun foo(a: <caret>) {}
```